### DISCUSSION TITLE
New Feature

### DISCUSSION THREAD

**Discussion by bonzoskill** [rohitsangwan01/uni_control_hub#28] [Ideas]
This program i think could be a better version (for the user experience) of this https://github.com/feschber/lan-mouse
Or at least yours could be more easy to setup for different devices. Anyway on Arch i had to use debtap to convert .deb package with the risk of the case.

**Comment by rohitsangwan01** [comment]
@bonzoskill LanMouse focuses on desktop connectivity, whereas this project is specifically aimed at mobile devices and my current goal is to keep the focus on improving mobile support.

and you can bundle the app in any format, Checkout [Developer's Guide](https://github.com/rohitsangwan01/uni_control_hub?tab=readme-ov-file#developers-guide) to install Flutter and then either run directly or pack the app with [Flutter Distributor](https://distributor.leanflutter.dev/)
